src/Language/DifferentialDatalog/OVSDB/Compile.hs

```diff
@@ -72,9 +73,11 @@ mkCol isinput tname c@TableColumn{..} = do
          ColumnTypeComplex ct -> mkComplexType isinput tname c ct
 
 mkColName :: String -> Doc
-mkColName "type" = "_type"
-mkColName "match" = "_match"
-mkColName n      = pp $ map toLower n
+mkColName x =
+    if elem x' reservedNames
+       then pp $ "__" ++ x'
```

Comment:
This is a bug waiting to happen - unless you reserve these identifiers.

Reply:
fixed